index.html

```diff
@@ -16,8 +16,13 @@ <h1>Timur Buchberger</h1>
+    <figure>
+        <img src="/img/Download.jpeg"
         alt="placeholder picture of a cute cat">
+        <br />
+        <figcaption>A placeholder picture of a beautiful cat</figcaption>
+    </figure>
```

Comment:
AWESOME! We can do the gap with CSS tomorrow, no need for `br` tags 🙂 
```suggestion
    <figure>
        <img src="/img/Download.jpeg"
        alt="placeholder picture of a cute cat">
        <figcaption>A placeholder picture of a beautiful cat</figcaption>
    </figure>
```

Reply:
Thank you :) Good to know...